Fix test cases styles

foriequal0 opened the pull request.

posgnu commented on tests/test_Uxxx.py:
```diff
@@ -1,13 +1,10 @@
 import inspect
 
 import pytest
-from rlp import decode
```
This is a result of `reorder_python_imports` which is used by flask project. I brought most of the formatting rules from this project. https://github.com/asottile/reorder_python_imports https://github.com/pallets/flask/blob/master/.pre-commit-config.yaml

posgnu commented:
LGTM

foriequal0 commented:
I've rebased it to `master` and added/fixed some commits.

* Use more flexible language_version: python3
* Fix `import` according to PEP8 style guide
* Fix EOL
* Fill default value for `__build__` in `__version__.py`
